scopesim_templates/metis/laser.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""WCU laser spectrum for METIS."""
+
+import numpy as np
+from astropy import units as u
+from astropy.io import fits
+
+from spextra import Spextrum
+
+from ..rc import Source, im_plane_utils
+
+
+def laser_spectrum(img_size=(12, 12),
+                   waves=np.arange(3, 6, .001)*u.um,
+                   centers=[3.39, 4.73, 5.263]*u.um,
+                   fwhms=[8E-8, 8E-7, 6E-5]*u.um,
+                   fluxes=[20, 1, 22]):
```

Comment:
It is bad form to put these lists here as defaults, because you'd have to trust that e.g. `Spextrum.emission_line_spectrum()` does not change the arguments. It is better to have `=None` here and then set the defaults in the function.

E.g. 
```python
def print_double_list(thelist=[1,2]):
    thelist *= 2
    print(thelist)

print_double_list()
print_double_list()
```
prints
```
[1, 2, 1, 2]
[1, 2, 1, 2, 1, 2, 1, 2]
```

It is not easy to see whether `Spextrum.emission_line_spectrum()` does manipulate the arguments. It passes the arguments to `flat_spectrum()`, `add_emi_lines()`, and `_angstrom_value()`, and other functions in `utils`. It seems all fine now, but maybe someone decides to change those functions later.